pkg/vm/memory/relocatable.go

```diff
@@ -62,6 +79,14 @@ func (m *MaybeRelocatable) GetRelocatable() (Relocatable, bool) {
 	return rel, is_type
 }
 
+func (m *MaybeRelocatable) IsZero() (is_zero bool) {
+	felt, is_zero := m.GetInt()
+	if is_zero {
+		is_zero = felt.Felt == 0
+	}
+	return
+}
```

Comment:
```suggestion
func (m *MaybeRelocatable) IsZero() bool {
	felt, is_int := m.GetInt()
	return is_int && felt.Felt == 0
}
```
I think variable `is_zero` should be used for one thing